fem/bilininteg_hcurl.cpp

```diff
@@ -1002,43 +1004,43 @@ void CurlCurlIntegrator::AssemblePA(const FiniteElementSpace &fes)
       {
          MFEM_VERIFY(coeffDim == dimc, "");
       }
-      if (MQ)
-      {
-         GM.SetSize(dimc);
-         MFEM_VERIFY(coeffDim == MQdim, "");
-         MFEM_VERIFY(MQ->GetHeight() == dimc && MQ->GetWidth() == dimc, "");
-      }
       if (SMQ)
       {
          SM.SetSize(dimc);
          MFEM_VERIFY(SMQ->GetSize() == dimc, "");
       }
+      else if (MQ)
+      {
+         M.SetSize(dimc);
```

Comment:
@pazner, it looks like we introduced a problem here. The `DenseMatrix` is named `GM` rather than `M` so this fails to compile. I'm not sure how we could have missed this. Perhaps a merge chose the wrong name...

Reply:
There is a member variable `M` that exists whenever `MFEM_THREAD_SAFE` is not defined.

Yes, this was caused by a merge with the shadow variables PR.